improver/cli/vicinity.py

```diff
@@ -66,10 +66,6 @@ def process(cube: cli.inputcube, vicinity: float):
         iris.cube.Cube:
             Cube with the vicinity processed data.
     """
-    from improver.metadata.probabilistic import in_vicinity_name_format
     from improver.utilities.spatial import OccurrenceWithinVicinity
 
-    result = OccurrenceWithinVicinity(radius=vicinity).process(cube)
-    result.rename(in_vicinity_name_format(result.name()))
-
-    return result
+    return OccurrenceWithinVicinity(radius=vicinity).process(cube)
```

Comment:
The acceptance test data for this CLI needs updating too?

Reply:
Yep, too many overlapping PRs and acceptance test data sets for me to keep straight apparently. Done.

Reply:
thanks